index.html

```diff
@@ -225,7 +225,7 @@ <h3 class="mbr-section-subtitle align-center mbr-light pb-3 mbr-fonts-style disp
           <a class="anchor" name="iris" id="iris"></a>
           <div class="card-wrapper">
             <div class="card-img">
-              <img src="/static/images/iris-1046x697.png"
+              <img src="/static/images/Iris7_1_trim_full.png"
```

Comment:
Note the new image (taken from https://github.com/SciTools/iris/tree/master/docs/iris/src/_static) is much higher resolution than the old image:
```
$ file Iris7_1_trim_full.png
Iris7_1_trim_full.png: PNG image data, 2085 x 2109, 8-bit/color RGBA, non-interlaced
```

Reply:
We could go for the smaller Iris7_1_trim_10.png but I was unsure if that would be too small??